runtime/src/accounts_db.rs

```diff
@@ -1104,10 +1112,8 @@ impl AccountsDB {
             start.stop();
             handle_reclaims_elapsed = start.as_us();
 
-            let mut start = Measure::start("write_storage_elapsed");
```

Comment:
@carllin oops. maybe unintentional remove? We're starting mixing the measured time for `"write_storage_elapsed"` with `"update_index_elapsed"` by removing `"write_storage_elapsed"`.

Reply:
ah good catch, thanks!